tests/test_encoding/test_similarity_encoder.py

```diff
@@ -0,0 +1,176 @@
+import pandas as pd
+import pytest
+from feature_engine.encoding import StringSimilarityEncoder
+
+
+def test_encode_top_categories():
```

Comment:
at the moment, we are testing with letters. Do I understand correctly that the distance will be 1 for each of them?

Should we not add a test with a more realistic scenario, to test that we are indeed using this similarity function from difflib? If someone changes this function for another similarity measure, this test will probably still pass?

Reply:
I don't think that we should test functions from standard library, that would be excessive